torch/linalg/__init__.py

```diff
@@ -140,6 +140,66 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+cond = _add_docstr(_linalg.linalg_cond, r"""
+linalg.norm(input, p=None, *, out=None) -> Tensor
+
+Returns the condition number of a matrix.
+The condition number of :attr:`input` is defined as the norm of
+:attr:`input` times the norm of the inverse of :attr:`input`.
+
+This function supports real-valued, and only on CPU, complex-valued input.
+
+Args:
+    input (Tensor): the input tensor of size :math:`(*, m, n)` where `*` is zero or more
+                    batch dimensions.
```

Comment:
It's not multiple dimensions, it's just that the batch may contain multiple matrices.